proposals/NNNN-PerformInteractionMultipick.md

```diff
@@ -0,0 +1,249 @@
+	  <element name="APPLICATION_NOT_REGISTERED"/>
+	  <element name="GENERIC_ERROR"/>
+	  <element name="REJECTED"/>
+	  <element name="INVALID_ID"/>
+	  <element name="DUPLICATE_NAME"/>
+	  <element name="TIMED_OUT"/>
+	  <element name="ABORTED"/>
+	  <element name="UNSUPPORTED_RESOURCE"/>
+	  <element name="WARNINGS"/>
+	</param>
+	
+	<param name="info" type="String" maxlength="1000" mandatory="false" platform="documentation" since="X.X">
+	  <description>Provides additional human readable info regarding the result.</description>
+	</param>
+	
+	<param name="choiceID" type="Integer" array="true" minlength="0" maxlength="500" minvalue="0" maxvalue="2000000000" mandatory="false" since="X.X">
+	  <description>
+		ID of the choice that was selected in response to PerformInteraction.
+		Only is valid if general result is "success:true".
+	  </description>
+	</param>
+  </function>
+```
+
+
+and `HMI_API`:
+```xml
+<interface name="UI" version="X.X.X" date="X-X-X">
+  <function name="PerformInteractionMultipick" messagetype="request">
+    <description>Triggers an interaction where a user can pick multiple things (e.g. "What do you want to order from this restaurant?"). </description>
+    <param name="initialText" type="Common.TextFieldStruct" mandatory="false">
+      <description>Uses initialInteractionText. See TextFieldStruct.</description>
+    </param>
+    <param name="multiPickConfirmationButtonText" type="String" maxlength="500" mandatory="false">
+      <description>
+        This text will be the button that a person uses to finish choosing items.
+      </description>
+    </param>
+    <param name="choiceSet" type="Common.Choice" minsize="1" maxsize="100" array="true" mandatory="false">
+      <description>The list of choices to be used for the interaction with the user</description>
+    </param>
+    <param name="timeout" type="Integer" minvalue="5000" maxvalue="100000" defvalue="10000" mandatory="true">
+      <description>Timeout in milliseconds.</description>
+    </param>
+    <param name="interactionLayout" type="Common.LayoutMode" mandatory="false">
+      <description>See LayoutMode.</description>
+    </param>
+    <param name="appID" type="Integer" mandatory="true">
+      <description>ID of application that concerns this RPC.</description>
+    </param>
+    <param name="cancelID" type="Integer" mandatory="false">
+      <description>
+        An ID for this specific alert to allow cancellation through the `CancelInteraction` RPC.
+      </description>
+    </param>
+  </function>
+</interface>
+```
+```xml
+  <function name="PerformInteractionMultipick" messagetype="response">
+    <param name="choiceID" type="Integer" minsize="1" maxsize="100" minvalue="0" maxvalue="2000000000" array="true" mandatory="false">
+      <description>IDs of the choices that were selected. Allows duplicate choiceIDs so that the HMI can send back the same choice ID multiple times to relay a quantity back to the app.

```

Comment:
```suggestion
      <description>IDs of the choices that were selected. Allows duplicate choiceIDs so that the HMI can send back the same choiceID multiple times to relay a quantity back to the app.

```